src/components/radioButtonGroupWithTitle.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactElement } from "react"
+
+import RadioButton, { RadioButtonProps } from "./radioButton"
+import Label from "./fieldHelpers/label"
+
+interface Props {
+  title: string
+  required?: boolean
+  disabled?: boolean
+  radioInputs: RadioButtonProps[]
+}
+
+function RadioButtonGroup({
+  title,
+  required = false,
+  disabled = false,
+  radioInputs,
+}: Props): ReactElement {
+  return (
+    <div className="w-12/12">
+      <Label fieldName={title} required={required} />
+      <div className="flex justify-between">
+        {radioInputs.map((radioInput, index) => (
+          <RadioButton
+            key={index}
+            name={radioInput.name}
+            onChange={radioInput.onChange}
+            value={radioInput.value}
+            checked={radioInput.checked}
+            renderLabel={radioInput.renderLabel}
+            disabled={disabled}
```

Comment:
is there a reason why it's not part of the radioInput object?

Reply:
Yes, it's because I think we should be able to set it up globally to disable every `RadioButtons` at once